### PR TITLE
[13.0][ADD] sale_product_variant_attribute_tax: New module

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT27131

Depends on:
- [ ] https://github.com/OCA/product-variant/pull/205

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (pedrobaeza) [inline .vscode/settings.json:1]: Remove this

File: sale_product_variant_attribute_tax/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2016-2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Put attribute taxes on sales orders",
+    "summary": "Bring the taxes associated to product values",
+    "version": "13.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "Sale",
```
Comment (pedrobaeza) [inline sale_product_variant_attribute_tax/__manifest__.py:8]: Is this the correct category (the same as `sale` module)?

File: sale_product_variant_attribute_tax/models/sale_order.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    _inherit = "sale.order.line"
+
+    @api.onchange("product_attribute_ids")
+    def _onchange_product_attribute_ids_configurator(self):
+        res = super()._onchange_product_attribute_ids_configurator()
+        # if there's a product set, taxes on that product have priority
+        if self.product_id or not self.product_tmpl_id:
+            return res
+        values = self.mapped("product_attribute_ids.value_id")
+        taxes = values.mapped("tax_ids")
```
Comment (pedrobaeza) [inline sale_product_variant_attribute_tax/models/sale_order.py:16]:
```suggestion
        taxes = values.mapped("tax_ids").filtered(lambda x: x.type_tax_use == "sale")
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
This should go to OCA/product-variant, as it depends on `sale_variant_configurator`.

**Comment by ernestotejeda** [comment]
@pedrobaeza changes done.
PR moved to https://github.com/OCA/product-variant/pull/207

**Comment by joao-p-marques** [comment]
> PR moved to OCA/product-variant#207

@ernestotejeda can you close this one then to avoid confusion?

**Pull request closed without merging**